### PR TITLE
Provide default link for quay clair vulnerabilities

### PR DESCRIPTION
**Pull request by mayaCostantini**
## Related Issues and Dependencies

Fixes https://github.com/thoth-station/prescriptions/issues/20956

## This introduces a breaking change

- No

## This should yield a new module release

- Yes

### CODE REVIEW

**Comment by mayaCostantini** [comment]
Ready for review, @fridex PTAL 👍🏻

File: thoth/prescriptions_refresh/handlers/quay/security.py [hunk 1]
```diff
@@ -165,11 +165,21 @@ def _create_vulnerability_prescriptions(image: str, tag: str, vulnerabilities: L
 
         cve_seen.add(vulnerability["Name"])
 
+        if "Link" in vulnerability.keys():
+            vulnerability_link = vulnerability["Link"]
+            message = f"Found vulnerability {vulnerability['Name']!r} in the base image used."
+            " See the link provided for more information."
+
```
[the remark this answers is not in the document]
Comment (mayaCostantini) [reply thoth/prescriptions_refresh/handlers/quay/security.py:172]: Looks like a good idea, we could have this in 👍🏻 I will create the corresponding justification and make the change here and on the prescriptions we already computed were we provided https://access.redhat.com/errata/#/ as default, thanks!

File: thoth/prescriptions_refresh/handlers/quay/security.py [hunk 2]
```diff
@@ -165,11 +165,21 @@ def _create_vulnerability_prescriptions(image: str, tag: str, vulnerabilities: L
...
+        else:
+            vulnerability_link = "https://thoth-station.ninja/j/no_cve_link_provided"
```
Comment (fridex) [inline thoth/prescriptions_refresh/handlers/quay/security.py:174]: You can directly use the following link that internally translates to our justifications:

```suggestion
            vulnerability_link = "no_cve_link_provided"
```

**Review by fridex — COMMENTED** [review]
lgtm, just few cosmetic things

**Review by fridex — APPROVED** [review]
lgtm, thanks! 💯

**Pull request merged**